scripts/compile-dots.js

```diff
@@ -3,8 +3,14 @@
 var glob = require('glob')
   , fs = require('fs')
   , path = require('path')
-  , doT = require('dot')
-  , beautify = require('js-beautify').js_beautify;
+  , doT = require('dot');
```

Comment:
As is, this script requires devDependencies and is not exposed via package.json's bin field. It's okay for this one to have a hard dependency on js-beautify.